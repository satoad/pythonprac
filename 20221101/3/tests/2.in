f, g, h = Grange(6, 3, 108575), Grange(7, 5, 104856), Grange(11, 11, 10485577)
a, b, c = Grange(12, 4, 1000), Grange(7, 5, 104856), Grange(5, 5, 10000)
print(f"{f=}, {g=}, {h=}")
print(f"{list(a)}({not(a)}), {list(c)}({not(c)})")
print(f"{len(f)=}, {len(g)=}, {len(h)=}")
print(f"{len(list(f))=}, {len(list(g))=}, {len(list(h))=}")
print(f"{f[2]=}, {h[33]=}")
one, two = f[3:3276], h[3:3267:5]
print(f"{one=}, {one[1]=}")
print(f"{two=}, {two[2]=}, {list(two)=}")
one, two = f[3:3276], b[1:1:1]
print(f"{one=}, {one[1]=}")
print(f"{two=}, {two[2]=}, {list(two)=}")