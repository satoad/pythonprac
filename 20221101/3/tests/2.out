f=grange(6, 3, 108575), g=grange(7, 5, 104856), h=grange(11, 11, 10485577)
[12, 48, 192, 768](False), [5, 25, 125, 625, 3125](False)
len(f)=9, len(g)=6, len(h)=6
len(list(f))=9, len(list(g))=6, len(list(h))=6
f[2]=54, h[33]=255476698618765889551019445759400441
one=grange(3, 3, 3276), one[1]=9
two=grange(3, 161051, 3267), two[2]=77812273803, list(two)=[3]
one=grange(3, 3, 3276), one[1]=9
two=grange(1, 5, 1), two[2]=25, list(two)=[]
